lib/improver/tests/nowcasting/nowcasting/test_ApplyOrographicEnhancement.py

```diff
@@ -0,0 +1,457 @@
+        """Test extracting a time coordinate from the orographic enhancement
+        cube."""
+        plugin = ApplyOrographicEnhancement("add")
+        result = plugin._extract_orographic_enhancement_cube(
+            self.precip_cubes[0], self.first_slice)
+        self.assertIsInstance(result, iris.cube.Cube)
+        self.assertEqual(result.metadata, self.first_slice.metadata)
+        self.assertEqual(result, self.first_slice)
+        self.assertEqual(result.coord("time"), self.first_slice.coord("time"))
+
+    def test_alternative_time_quarter_past(self):
+        """Test extracting a time coordinate from the orographic enhancement
+        cube at quarter past an hour."""
+        plugin = ApplyOrographicEnhancement("add")
+        self.precip_cubes[0].coord("time").points = 412227.25
+        result = plugin._extract_orographic_enhancement_cube(
+            self.precip_cubes[0], self.oe_cube)
+        self.assertIsInstance(result, iris.cube.Cube)
+        self.assertEqual(result.metadata, self.first_slice.metadata)
+        self.assertEqual(result, self.first_slice)
+        self.assertEqual(result.coord("time"), self.first_slice.coord("time"))
+
+    def test_alternative_time_half_past(self):
+        """Test extracting a time coordinate from the orographic enhancement
+        cube at half past an hour."""
+        plugin = ApplyOrographicEnhancement("add")
+        self.precip_cubes[0].coord("time").points = 412227.51
+        result = plugin._extract_orographic_enhancement_cube(
+            self.precip_cubes[0], self.oe_cube)
+        self.assertIsInstance(result, iris.cube.Cube)
+        self.assertEqual(result.metadata, self.second_slice.metadata)
+        self.assertEqual(result, self.second_slice)
+        self.assertEqual(result.coord("time"), self.second_slice.coord("time"))
+
+    def test_alternative_time_quarter_to(self):
+        """Test extracting a time coordinate from the orographic enhancement
+        cube at quarter to an hour."""
+        plugin = ApplyOrographicEnhancement("add")
+        self.precip_cubes[0].coord("time").points = 412227.75
+        result = plugin._extract_orographic_enhancement_cube(
+            self.precip_cubes[0], self.oe_cube)
+        self.assertIsInstance(result, iris.cube.Cube)
+        self.assertEqual(result.metadata, self.second_slice.metadata)
+        self.assertEqual(result, self.second_slice)
+        self.assertEqual(result.coord("time"), self.second_slice.coord("time"))
+
+
+class Test__apply_cube_combiner(IrisTest):
+
+    """Test the __apply_cube_combiner method."""
+
+    def setUp(self):
+        """Set up cubes for testing."""
+        self.precip_cubes = set_up_precipitation_rate_cube()
+        self.oe_cubes = set_up_orographic_enhancement_cube()
+
+    def test_check_expected_values_first_add(self):
+        """Test the expected values are returned when cubes are added.
+        First check."""
+        expected0 = np.array([[[[0., 1., 2.],
```

Comment:
Can we lose the 0 from the variable name please.

Reply:
Removed 0 from variable name and from all other variable names within this file where it is not required.